.circleci/config.yml

```diff
@@ -147,7 +147,10 @@ jobs:
       - run:  # Fail audit only for moderate and higher vulnerabilities
           name: Audit vulnerability of backend node_modules
           command: |
-            yarn audit --level low; EXIT_CODE=$?; \
+            cd frontend
```

Comment:
This is supposed to be the audit for the BE, you're now auditing the FE twice. Wouldn't we want to run that script both for the FE and the BE deps audit?
```suggestion
```

Reply:
Aaaand, apparently @rahearn and I were writing the same thing at the same time. Whoops.